src/NUnitEngine/nunit.engine.core/RuntimeFramework.cs

```diff
@@ -641,6 +684,73 @@ private static void AddMonoFramework(Version frameworkVersion, string profile)
 
             _availableFrameworks.Add(framework);
         }
+
+        private static void FindDotNetCoreFrameworks()
+        {
+            const string WINDOWS_INSTALL_DIR = "C:\\Program Files\\dotnet\\";
```

Comment:
My understanding is that this is a default installation location, but not guaranteed, users can install where they please. 

As an alternative/additional implementation to this method, how about invoking `dotnet --list-runtimes`? The output from that includes paths to exe's.

I'm don't know where that command gets its info from. Although does seem like an issue we can then delegate to dotnet itself

Reply:
I used this approach because I didn't want to run the dotnet command without first knowing that dotnet was installed. I'll look into this further. At the time I wrote it I did a bunch of research, but I've forgotten the details. I'll check further.

Reply:
I think https://github.com/nunit/nunit-console/pull/937#discussion_r623511925 refers to this thread. Agree about separating this out from this PR, I'll create a new thread here: https://github.com/nunit/nunit-console/pull/937#discussion_r623511925